include/library/spdm_requester_lib.h

```diff
@@ -328,6 +330,8 @@ libspdm_return_t libspdm_start_session(void *spdm_context, bool use_psk,
  * @retval RETURN_SECURITY_VIOLATION    Any verification fails.
  **/
 libspdm_return_t libspdm_start_session_ex(void *spdm_context, bool use_psk,
+                                          const void *psk_hint,
```

Comment:
These parameters need to be added to the Doxygen comment above. I can update the API documentation.

Reply:
Added.